_includes/header.html

```diff
@@ -0,0 +1,7 @@
+<!-- Not currently in use -->
+<!-- Header component -->
+<header>
+    <nav>
+        <!-- Add navigation items here -->
+    </nav>
+</header>
```

Comment:
suggest removing empty/unused files

Reply:
Was left as we spoke q while ago about potentially adding more pages which we would need a nav bar for. Happy to delete if this isn't happening anytime soon @petebankhead?

Reply:
No strong opinions from me. This file would help me know where to begin putting a nav bar, but I'm hoping that I never have to do it myself.

Reply:
Fine with me to just remove the commented `{% include header.html %}` but equally happy to act as jekyll consultant/scapegoat if we do end up needing to add a navbar later

Reply:
it has now been removed, thanks!